client/deis.py

```diff
@@ -789,8 +790,11 @@ def auth_login(self, args):
             provide a username for the account.
           --password=<password>
             provide a password for the account.
+          --ssl-verify=false
+            disables ssl certificate verification for API requests
```

Comment:
Could you change this to "SSL" (capitalized) here? Sorry for the nitpick! We just reviewed this and it's good to go otherwise.